tests/storage/test_write.py

```diff
@@ -0,0 +1,79 @@
+import hashlib
+import os
+import glob
+import traceback
+import re
+import binascii
+import logging
+import time
+from pathlib import Path
+from IOUtils import ReadWriteOp
+import sys
+
+sys.path.append("..")
+from logger import Logger, LOG_FOLDER, LOG_EXTENSION
+from TestBase import Test
+import argparse
+
+extension = ".txt"
```

Comment:
I have removed it from other files except `test_checksum.py` and `test_throughput.py`

Reply:
OK